be/src/exprs/utility_functions.cpp

```diff
@@ -39,4 +39,82 @@ BooleanVal UtilityFunctions::sleep(FunctionContext* ctx, const IntVal& seconds)
     return BooleanVal(true);
 }
 
+static void convert_to_string(doris_udf::StringVal& src, std::string& str_dst) {
```

Comment:
why called `convert_to_string` ？ just a concat ?

Reply:
just want to convert the type from 'StringVal' to 'string'.